syft/artifact/relationship.go

```diff
@@ -15,6 +15,9 @@ const (
 
 	// DescribedByRelationship is a proxy for the SPDX 2.2.2 DESCRIBED_BY relationship.
 	DescribedByRelationship RelationshipType = "described-by"
+
+	// DescribesRelationship is a proxy for the SPDX 2.3.0 DESCRIBES relationship.
+	DescribesRelationship RelationshipType = "describes"
```

Comment:
I've been trying to keep all of the relationships that syft outputs "one-way" by design and not express "both ways". That is, we have a `described-by` relationship which `describes` expresses the same edge type in a different direction. Can we use the existing `described-by` relationship we have today (and flip the node order in the relationship) instead of duping the kind in order to express a different direction?

Reply:
Great question! I think we can do that on the syft side and then have some formatting specific logic in the spdx format model code that gets the `DESCRIBES` wording into the final document.

^ Does this work for you? Apologies for including one-way edges that mirrored effort here - tunnel-visioned on the validation a little too hard.

Reply:
Removed